assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -159,12 +160,29 @@ export const selectors = {
 	 * @since 1.7.0
 	 *
 	 * @param {Object} state Data store's state.
+	 * @param {string} page Page query argument to add to admin URL.
+	 * @param {Object} args Additional query arguments to add to admin URL.
```

Comment:
Similarly, this should use `{(Object|undefined)}`, and also provide the "Optional. ".

Reply:
:+1: thanks.